object A not found, ERROR!ERROR in base instance_reference!  [5:21]
